### DISCUSSION TITLE
Permission storage.buckets.list required to access bucket in Google  Cloud Storage

### DISCUSSION THREAD

**Discussion by tomboxmeme** [iterate-ch/cyberduck#16315] [Q&A]
Hi all,

Running OSX and Cyberduck 9.0.1

I'm trying to add perms to a specific folder in GCS object storage but for some reason Cyberduck keeps throwing the error that it's looking for `storage.buckets.list`. 

I obviously don't want the user to be able to list all buckets so is there something I'm missing? 

It seems to completely ignore the path variable.

**Comment by dkocher** [comment]
This is reproducible and occurs because we attempt to read the bucket versioning configuration prior listing files.

```
… does not have storage.buckets.list access to the Google Cloud project. Permission 'storage.buckets.list' denied on resource (or it may not exist). Domain: global. Reason: forbidden. Forbidden. Please contact your web hosting service provider for assistance.
```

You will have to ensure the IAM user has a role assigned that includes `storage.buckets.list `.

**Reply by dkocher** [reply]
We will relax the requirements for permissions with #16469.

**Comment by jackgle** [comment]
This issue still exists (at least in Mac version), and there is no Path variable.

**Reply by dkocher** [reply]
You can find the _Path_ setting when [editing the bookmark](https://docs.cyberduck.io/cyberduck/bookmarks/#edit-bookmark).

**Reply by jackgle** [reply]
Awesome, thanks!
